backend/src/controller/base_layers.rs

```diff
@@ -0,0 +1,61 @@
+//! `BaseLayer` endpoints.
+
+use crate::{db::connection::Pool, service};
+use actix_web::{
+    delete, get, post,
+    web::{Data, Json, Path},
+    HttpResponse, Result,
+};
+
+use crate::model::dto::NewBaseLayerDto;
+
+/// Fetch a single [`BaseLayerDto`](crate::model::dto::BaseLayerDto) by its id.
+///
+/// # Errors
+/// * If the connection to the database could not be established.
+#[utoipa::path(
+    context_path = "/api/layers/base/{id}",
```

Comment:
Remove `/{id}` here. It should just be `context_path = "/api/layers/base`.
Context path is simply the thing utoipa cannot infer from the function itself (meaning the rest of the path that is defined in routes)

Reply:
Still open

Reply:
Why is this marked as resolved?
There were no changes not even an explanation as to why it is supposed to be resolved.